include/PluginFactory.h

```diff
@@ -85,6 +88,7 @@ public slots:
 	DescriptorMap m_descriptors;
 	PluginInfoList m_pluginInfos;
 	QMap<QString, PluginInfo> m_pluginByExt;
+	QVector<std::string> m_garbage; //!< cleaned up at destruction
```

Comment:
I used this in PluginFactory.cpp: The unique name must be constructed as std::string `std::string unique_name = spa::unique_name(*descriptor);`, but the `Plugin::Descriptor` expects a C string. So this can be solved by giving `Plugin::Descriptor` a reference to a C string stored in an std::string, whereas that std::string is
* on stack which we remove later (bad)
* simply new allocated on heap (will work, but it's a memleak)
* inside a garbage vector

I didn't see any better solution.